media/transcoder/job_test.py

```diff
@@ -178,7 +179,7 @@ def test_create_job_from_ad_hoc(capsys, test_bucket):
 
     time.sleep(30)
 
-    _assert_job_state_succeeded(capsys, job_id)
+    _assert_job_state_succeeded_or_running(capsys, job_id)
```

Comment:
Forgive me for my lack of expertise in this, what happens if the job is still running but we run the code below? (list jobs, delete job). Would they still run as expected: jobs are listed fine, and the running job is paused and deleted accordingly?

Reply:
`_assert_job_state_succeeded_or_running` now tests if the job is running or has completed (rather than just completed). Sometimes jobs get stuck in the RUNNING state and won't complete for days, possibly failing along the way. This led to flaky tests. The tradeoff here is that the API is not being tested end to end by this test, but this isn't necessary anyway for the code samples.

Regardless, as long as the list function comes before the delete function, the test should run as expected. It is ok to list or delete a job that is running or not running (not started, failed, or completed).

Reply:
Sounds good! Thank you for the explanation for this :)

Could you add the bit about this being ok to list or delete a job that is running as a comment? Just having it on `_assert_job_state_succeeded_or_running` function's comment will help let other folks know about this unique situation.

Reply:
Done